src/components/SudokuGrid/Row.js

```diff
@@ -0,0 +1,29 @@
+import Cell from "./Cell";
+import { v4 as uuidv4 } from "uuid";
+
+export default function Row({
+  currentSudokuNumbers,
+  rowNumber,
+  mask,
+  onCellInRowClick,
+}) {
+  return (
+    <div className="Sudoku__row">
+      {currentSudokuNumbers[rowNumber].map((number, columnNumber) => {
+        let editable = true;
+        if (mask[rowNumber][columnNumber] !== 0) {
+          editable = false;
+        }
+        return (
+          <Cell
+            key={uuidv4()}
+            value={number}
+            id={[rowNumber, columnNumber]}
```

Comment:
This is a weird looking `id`.

Reply:
Hahah, yeah. Would it be bad practice to leave it like that, have an array as the id?